bin/sozo/src/commands/auth.rs

```diff
@@ -102,14 +108,25 @@ pub async fn grant(
     kind: AuthKind,
     transaction: TransactionOptions,
 ) -> Result<()> {
+    trace!(target: LOG_TARGET, ?kind, ?world, ?starknet, ?account, ?transaction, "Executing 'Grant' command.");
```

Comment:
```suggestion
    trace!(target: LOG_TARGET, ?kind, ?world, ?starknet, ?account, ?transaction, "Executing Grant command.");
```
Without quote to be consistent with what you already have.